implementations/rust/ockam/ockam_api/src/authenticator/storage/authority_member.rs

```diff
@@ -0,0 +1,79 @@
+use ockam::identity::{Identifier, TimestampInSeconds};
+use ockam_core::compat::collections::BTreeMap;
+use ockam_core::compat::str::FromStr;
+use ockam_core::{Error, Result};
+
+/// Project member stored on the Authority node
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct AuthorityMember {
```

Comment:
Still wondering about the naming. The name is `AuthorityMember` but all the comments talk about a "project member". I would call simply call this entity a `Member` (which kind of begs the question "a member of what?". The what is not really encoded here, maybe only in `added_by`).

Reply:
I think that's because the only Authority we now have is the Project Authority. Let's make it clearer when new ones appear?

Reply:
That's ok.